src/axom/sidre/examples/CMakeLists.txt

```diff
@@ -88,6 +89,17 @@ if(ENABLE_MPI)
                            )
         endif()
     endforeach()
+
+    blt_add_test( NAME sidre_createdatastore_parallel
```

Comment:
Should we also test these with a single rank, i.e. `NUM MPI_TASKS 1`?

Reply:
Single rank tests for these are created in the foreach loop immediately above this.

Reply:
Thanks @nselliott -- hard to see from github :)